test/dynamo/test_flat_apply.py

```diff
@@ -83,6 +84,41 @@ def f(a, b):
         result = flat_apply(func_spec, in_spec, *flat_args)
         self.assertEqual(result, f(*args, **kwargs))
 
+    def test_mark_traceable_dynamo_graph(self):
+        @torch._dynamo.mark_traceable
+        def func(x, y, z):
+            torch._dynamo.graph_break()
+            return x * y + z
+
+        backend = EagerAndRecordGraphs()
+
+        @torch.compile(fullgraph=True, backend=backend)
+        def fn(x, y):
+            t0 = x + 1
+            t1 = func(x, y, t0)
```

Comment:
I'll update this example to include a pytree-registered input to make it less boring.